packages/steveo/src/lib/app-error.ts

```diff
@@ -0,0 +1,7 @@
+import { LogEntry, Logger } from './logger';
+
+export class AppError {
+  constructor(logger: Logger, public obj: LogEntry) {
+    logger.error(obj);
```

Comment:
Not sure i am sold on an error abstraction. 
- For consistency, i think we should introduce it as a pattern if we think there's value
- Without enforcement, these abstractions tend to get neglected and wither away.
Thoughts?

Reply:
This is an experiment to see how it tracks
The intention is to elevate it to a library if it works/is accepted